spec/controllers/posts_controller_spec.rb

```diff
@@ -638,6 +661,10 @@
       expect(PostTag.count).to eq(0)
     end
 
+    it "handles invalid first reply" do
+      # TODO: Add a test for this if it becomes possible to have an invalid reply here
```

Comment:
This should explicitly say `skip` in it, as currently it'll act as a passing test.